app/views/rooms/location.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= provide(:title, "Location") %>
+<div class="row mx-4">
+    <div class="col-2">
+        <%= render 'rooms/menu' %>
+    </div>
+    <div class="col-10">       
+        <div class="card container d-flex justify-content-center mt-4">
+            <div class="card-header">
+                <h3>Location<h3>
+            </div>
+            <%= form_for(@room) do |f| %>
+            <div class="card-body">
+                <div class="form-group mx-4">
+                    <%= f.label :Address %>
+                    <%= f.text_field :is_active, class: "form-control" %>
+                </div>
+            </div>
+            <div class="text-center pb-4">
+                <%= f.submit "Update Location", class: "btn btn-outline-dark" %>
```

Comment:
we will be implementing the one function from the gem we installed during setup,

make sure you have the address field in the migrations first

please follow this discussion slides to implement the necessary feature:
https://qiita.com/tiara/items/573fe5f1a84ca57dabcd

if you have any questions for setting up geocoder please inform me